core/flyout_horizontal.js

```diff
@@ -245,16 +245,60 @@ Blockly.HorizontalFlyout.prototype.scrollToStart = function() {
 };
 
 /**
+ * Scroll the flyout to a position.
+ * @param {Number} pos The targeted scroll position.
+ */
+Blockly.HorizontalFlyout.prototype.scrollTo = function(pos) {
+  this.scrollTarget = pos * this.workspace_.scale;
+
+  // Make sure not to set the scroll target below the lowest point we can
+  // scroll to, i.e. the content height minus the view height
+  var contentWidth = this.workspace_.getMetrics().contentWidth;
+  var viewWidth = this.workspace_.getMetrics().viewWidth;
+  this.scrollTarget = Math.min(this.scrollTarget, contentWidth - viewWidth);
+
+  this.step();
+};
+
+/**
+ * Step the scrolling animation by scrolling a fraction of the way to
+ * a scroll target, and request the next frame if necessary.
+ */
+Blockly.HorizontalFlyout.prototype.step = function() {
```

Comment:
This looks like a private function.  Annotate it as such, and add an underscore to the end of the name.

Reply:
yep